discovery-provider/src/monitors/memory.py

```diff
@@ -0,0 +1,11 @@
+import psutil
+
+
+def get_total_memory(**kwargs):
```

Comment:
i think it's probably worth having some comments explaining these monitors, at the very least what they are returning and why - maybe not in every function but somewhere